packages/announcement-controller/src/AnnouncementController.ts

```diff
@@ -122,6 +122,21 @@ export class AnnouncementController extends BaseController<
     });
   }
 
+  /**
+   * Resets the isShown status for all announcements
+   */
+  resetIsShownStates(): void {
```

Comment:
This function name is very literal. That makes sense, but I'm curious perhaps there is a more natural-sounding name that matches the other methods. Perhaps `hideAllAnnouncements`?

Reply:
I like the idea of a more natural-sounding name. 

Initially I named this `resetViewed` to resemble `updateViewed` below. possibly we use `resetViewed`? Wording with logic is a bit odd here as I think `isShown` is meant to represent `hasShown`  which would mean `hideAllAnnouncements` doesn't actively hide all the announcements but rather sets it to un-shown. thoughts ?

Reply:
@mcmire

Reply:
updated to `resetViewed` and happy to update again if we'd like to change it again [fc3b41f](https://github.com/MetaMask/core/pull/4088/commits/fc3b41f5da51bfe57541189edb242afaca91ee52)

Reply:
updated to `resetViewed` and happy to update again if we'd like to change it again [fc3b41f](https://github.com/MetaMask/core/pull/4088/commits/fc3b41f5da51bfe57541189edb242afaca91ee52) 

cc: @mcmire

Reply:
Ah, okay, I wasn't aware there was a subtle difference between shown and viewed. Curious if that's worth simplifying in the future. In any case, your reasoning makes sense.